developer/js/tests/test-join-word-breaker.ts

```diff
@@ -1,29 +1,30 @@
 var assert = require('chai').assert;
 
-var defaultWordBreaker = require('../../build/intermediate').wordBreakers['default'];
-var decorateWithJoin = require('../../build/intermediate').wordBreakers['join_'];
+var defaultWordBreaker = require('../../../common/predictive-text/build/intermediate').wordBreakers['default'];
+import {decorateWithJoin} from '../dist/lexical-model-compiler/join-word-breaker-decorator';
```

Comment:
Haven't these now been moved completely?

Reply:
I haven't yet merged master into this branch.

Reply:
Update: _that_ PR hasn't been merged into master yet! (see #3128)

Reply:
?  I thought "_that_ PR" was #3125, which has been merged to master.  This comment is on word-breaker `require`s, not template `require`s.

Reply:
Haha, good point! Derp. I'll fix it!